ArduPlane/GCS_Plane.cpp

```diff
@@ -8,17 +8,8 @@ uint8_t GCS_Plane::sysid_this_mav() const
 
 void GCS_Plane::update_vehicle_sensor_status_flags(void)
 {
-    // airspeed
-    const AP_Airspeed *airspeed = AP_Airspeed::get_singleton();
```

Comment:
Note I changed the logic on this bit so we don't check nullptr & airspeed->enabled() more than necessary.